applications/conceptual-model-editor/src/app/diagram/reactflow/components/menu-options.tsx

```diff
@@ -1,12 +1,14 @@
-import { useEffect, useRef, useState } from "react";
+import React, { useEffect, useRef, useState } from "react";
 
 export const useMenuOptions = () => {
     const [isMenuOptionsOpen, setIsMenuOptionsOpen] = useState(false);
     const menuOptionsRef = useRef<HTMLDivElement | null>(null);
 
-    const close = (e: React.MouseEvent | React.FocusEvent) => {
+    const close = (e?: React.MouseEvent | React.FocusEvent) => {
```

Comment:
Sinve you are changing this .. again get rid of single character variable names. Use "event" instead of "e". Here it may not matter, but on other places it does and it is easier to have a simple rule.

Reply:
Ok, I will try to remember to change single character variables if I see some. 

About the single characters, if I have something like:

function getRemainingNodes(nodes: Node[], node: Node) {
 return nodes.filter(n => n.id !== node.id);
}

Is the single character here ok, or should I also try to avoid it by for example renaming the arguments of function?